reunioes/progressos/intermediario/php_oo/metodos/Lutador.php

```diff
@@ -0,0 +1,38 @@
+<?php
+    class Lutador{
+        private $nome;
+        private $modalidade;
+
+        public function __construct($fighter, $modality){
+            $this->nome = $fighter;
+            $this->modalidade = $modality;
+        }
+
+        public function gritoDeGuerra($lutador){
+            foreach ($lutador as $key) {
+                if($key == "Julio"){
+                    return  "</br> Grito de guerra: " . "birrrrllll";
+                }else{
+                    return "</br> Grito de guerra: " . "(voz fina) Aqui é Brasil!!!";
+                }
+           }
+        }
+
+        public function __toString(){
+            return $this->nome . ", " . $this->modalidade;
+        }
+
+         public function __destruct(){
+            echo "</br>" . "Luta finalizada";
```

Comment:
Dê preferência por aspas simples! Vide notas anteriores.
Não quebre sua string quando for pequena, mantenha ma mesma instrução, exemplo:

```php
echo '</br>Luta finalizada';
```